zeno/classes/report.py

```diff
@@ -22,9 +22,11 @@ class Parameters(CamelModel):
 
 
 class Report(CamelModel):
+    id: int
     name: str
     type: Union[ChartType, None] = None
     slices: Union[List[str], None] = None
     metrics: Union[List[str], None] = None
     models: Union[List[str], None] = None
     parameters: Union[Parameters, None] = None
+    lastedit_at: str
```

Comment:
This should be `last_edited`

If we don't need an ID column I would rather not change the Report specification for this PR so we don't have to redo all the charts.

Reply:
hmm, I don't think we need an ID column, but to ensure the unique of naming, can we show a naming popup when hitting the "new chart" button?